### PR TITLE
german README.de.md

### PR DESCRIPTION
**Pull request by Spleezzer**
german translation

#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

I have created a german translation for the README.md

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: doc/README.de.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
+## Installation und Aktualisieungen
+Du kannst NewPipe auf die folgenden Weisen installieren:
+ 1. Füge ein eigenes Repository bei F-Droid hizu. Die Anweisungen findest du hier: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
+ 2. Lade die APK von [GitHub Releases](https://github.com/TeamNewPipe/NewPipe/releases) herunter und installiere sie.
+ 3. Aktualisierungen mit F-Droid. Dies ist die langsamste von allen Methoden, da F-Droid die Änderungen erkennen, die APK von selbst erstellen, unterzeichnen, und dann für die Benutzer veröffentlichen muss.
+ 4. Erstelle eine Debug-APK für dich. Dies ist die schnellste Methode, um neue Aktualisierungen zu bekommen, aber auch die komplizierteste. Wir empfehlen daher eine der anderen Methoden.
```
Comment (litetex) [inline doc/README.de.md:93]:
```suggestion
 4. Erstelle selber eine Debug-APK. Dies ist die schnellste Methode um neue Aktualisierungen zu bekommen, aber auch die komplizierteste. Wir empfehlen daher eine der anderen Methoden.
```

File: doc/README.de.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+* Bandcamp \[beta\]
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
+## Installation und Aktualisieungen
+Du kannst NewPipe auf die folgenden Weisen installieren:
+ 1. Füge ein eigenes Repository bei F-Droid hizu. Die Anweisungen findest du hier: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
+ 2. Lade die APK von [GitHub Releases](https://github.com/TeamNewPipe/NewPipe/releases) herunter und installiere sie.
+ 3. Aktualisierungen mit F-Droid. Dies ist die langsamste von allen Methoden, da F-Droid die Änderungen erkennen, die APK von selbst erstellen, unterzeichnen, und dann für die Benutzer veröffentlichen muss.
```
Comment (litetex) [inline doc/README.de.md:92]:
```suggestion
 3. Aktualisierung via F-Droid Dies ist die langsamste von allen Methoden, da F-Droid die Änderungen erkennen, die APK von selbst erstellen, signieren und dann für die Benutzer veröffentlichen muss.
```

File: doc/README.de.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+* media.ccc.de \[beta\]
+* PeerTube instances \[beta\]
+* Bandcamp \[beta\]
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
+## Installation und Aktualisieungen
+Du kannst NewPipe auf die folgenden Weisen installieren:
+ 1. Füge ein eigenes Repository bei F-Droid hizu. Die Anweisungen findest du hier: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
```
Comment (litetex) [inline doc/README.de.md:90]:
```suggestion
 1. Füge unser Repository bei F-Droid hinzu. Die Anweisungen gibt es hier: https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/
```

File: doc/README.de.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+* Warteschlangen in Playlists
+* Warteschlangen
+* Lokale Playlisten
+* Untertitel
+* Livestream Unterstützung
+* Kommentare werden angezeigt
+
+### Unterstützte Dienste
+
+NewPipe unterstützt diverse Dienste. Unsere [docs](https://teamnewpipe.github.io/documentation/) Unsere Dokumente enthalten weitere Informationen darüber, wie ein neuer Dienst zur App und zum Extraktor hinzugefügt werden kann. Bitte kontaktiere uns, wenn du einen Neuen hinzufügen möchtest. Die aktuell unterstützten Dienste sind:
```
Comment (litetex) [inline doc/README.de.md:77]:
```suggestion
NewPipe unterstützt diverse Dienste. Unsere [Dokumentation](https://teamnewpipe.github.io/documentation/) enthält weitere Informationen darüber, wie ein neuer Dienst zur App und zum Extraktor hinzugefügt werden kann. Bitte kontaktiere uns, wenn du einen Neuen hinzufügen möchtest. Die aktuell unterstützten Dienste sind:
```

File: doc/README.de.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+* 1080p/2K/4K Unterstützung
+* Verlauf
+* Abonniere Kanäle
+* Suche und abspielen von Playlists
+* Warteschlangen in Playlists
+* Warteschlangen
+* Lokale Playlisten
+* Untertitel
+* Livestream Unterstützung
+* Kommentare werden angezeigt
```
Comment (litetex) [inline doc/README.de.md:73]:
```suggestion
* Kommentare
```

File: doc/README.de.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+* Durchsuche Kanäle
+* Schaue Videos von einem Kanal aus
+* Orbot/Tor Unterstützung (noch nicht direkt)
+* 1080p/2K/4K Unterstützung
+* Verlauf
+* Abonniere Kanäle
+* Suche und abspielen von Playlists
+* Warteschlangen in Playlists
+* Warteschlangen
+* Lokale Playlisten
```
Comment (litetex) [inline doc/README.de.md:70]:
```suggestion
* Lokale Playlists
```

File: doc/README.de.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+* Zeigt allgemeine Informationen über einen Kanal an
+* Durchsuche Kanäle
+* Schaue Videos von einem Kanal aus
+* Orbot/Tor Unterstützung (noch nicht direkt)
+* 1080p/2K/4K Unterstützung
+* Verlauf
+* Abonniere Kanäle
+* Suche und abspielen von Playlists
+* Warteschlangen in Playlists
+* Warteschlangen
```
Comment (litetex) [inline doc/README.de.md:69]:
```suggestion
* Videos hintereinander wiedergeben
```

File: doc/README.de.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+* Durchsuche YouTube in verschiedenen Sprachen
+* Schaue/Blockiere altersbeschränkte Videos
+* Zeigt allgemeine Informationen über einen Kanal an
+* Durchsuche Kanäle
+* Schaue Videos von einem Kanal aus
+* Orbot/Tor Unterstützung (noch nicht direkt)
+* 1080p/2K/4K Unterstützung
+* Verlauf
+* Abonniere Kanäle
+* Suche und abspielen von Playlists
```
Comment (litetex) [inline doc/README.de.md:67]:
```suggestion
* Suchen und abspielen von Playlists
```

File: doc/README.de.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+* Wählen Sie den Streaming-Player aus, mit dem Sie Videos ansehen möchten
+* Lade Videos herunter
+* Lade nur den Ton herunter
+* Öffne ein Video in Kodi
+* Zeigt vorgeschlagene Videos an
+* Durchsuche YouTube in verschiedenen Sprachen
+* Schaue/Blockiere altersbeschränkte Videos
+* Zeigt allgemeine Informationen über einen Kanal an
+* Durchsuche Kanäle
+* Schaue Videos von einem Kanal aus
```
Comment (litetex) [inline doc/README.de.md:62]:
```suggestion
* Schaue Videos von einem Kanal an
```

File: doc/README.de.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+<hr>
+<p align="center"><a href="#screenshots">Screenshots</a> &bull; <a href="#description">Description</a> &bull; <a href="#features">Fuktionen</a> &bull; <a href="#installation-and-updates">Installation und Aktualisierungen</a> &bull; <a href="#contribution">Beitrag</a> &bull; <a href="#donate">Spende</a> &bull; <a href="#license">Lizens</a></p>
+<p align="center"><a href="https://newpipe.net">Webseite</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Presse</a></p>
+<hr>
+
+*Lese dieses Dokument in anderen Sprachen: [English](README.md), [Deutsch](doc/README.de.md), [Español](doc/README.es.md), [한국어](doc/README.ko.md), [Soomaali](doc/README.so.md), [Português Brasil](doc/README.pt_BR.md), [Polski](doc/README.pl.md), [日本語](doc/README.ja.md), [Română](doc/README.ro.md), [Türkçe](doc/README.tr.md), [正體中文](doc/README.zh_TW.md).*
+
+<b>WARNUNG: DIES IST EINE BETA-VERSION, DAHER KÖNNEN BUGS AUFTRETEN. WENN DU AUF WELCHE TRIFFST, ÖFFNE EIN ISSUE AUF DIESEM GITHUB REPOSITORY.</b>
+
+<b>DAS HOCHLADEN VON NEWPIPE VERSTÖSST GEGEN DIE ALLGEMEINEN GESCHÄFTSBEDINGUNGEN.</b>
```
Comment (litetex) [inline doc/README.de.md:24]:
```suggestion
<b>DAS HOCHLADEN VON NEWPIPE VERSTÖẞT GEGEN DIE NUTZUNGSBEDINGUNGEN.</b>
```

File: doc/README.de.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+</p>
+<hr>
+<p align="center"><a href="#screenshots">Screenshots</a> &bull; <a href="#description">Description</a> &bull; <a href="#features">Fuktionen</a> &bull; <a href="#installation-and-updates">Installation und Aktualisierungen</a> &bull; <a href="#contribution">Beitrag</a> &bull; <a href="#donate">Spende</a> &bull; <a href="#license">Lizens</a></p>
+<p align="center"><a href="https://newpipe.net">Webseite</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Presse</a></p>
+<hr>
+
+*Lese dieses Dokument in anderen Sprachen: [English](README.md), [Deutsch](doc/README.de.md), [Español](doc/README.es.md), [한국어](doc/README.ko.md), [Soomaali](doc/README.so.md), [Português Brasil](doc/README.pt_BR.md), [Polski](doc/README.pl.md), [日本語](doc/README.ja.md), [Română](doc/README.ro.md), [Türkçe](doc/README.tr.md), [正體中文](doc/README.zh_TW.md).*
+
+<b>WARNUNG: DIES IST EINE BETA-VERSION, DAHER KÖNNEN BUGS AUFTRETEN. WENN DU AUF WELCHE TRIFFST, ÖFFNE EIN ISSUE AUF DIESEM GITHUB REPOSITORY.</b>
```
Comment (litetex) [inline doc/README.de.md:22]:
```suggestion
<b>WARNUNG: DIES IST EINE BETA-VERSION, DAHER KÖNNEN UNERWARTETE PROBLEME AUFTRETEN. WENN DU AUF WELCHE TRIFFST, ÖFFNE EIN ISSUE IN DIESEM GITHUB REPOSITORY.</b>
```

File: doc/README.de.md [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+<a href="https://www.gnu.org/licenses/gpl-3.0" alt="License: GPLv3"><img src="https://img.shields.io/badge/License-GPL%20v3-blue.svg"></a>
+<a href="https://github.com/TeamNewPipe/NewPipe/actions" alt="Build Status"><img src="https://github.com/TeamNewPipe/NewPipe/workflows/CI/badge.svg?branch=dev&event=push"></a>
+<a href="https://hosted.weblate.org/engage/newpipe/" alt="Translation Status"><img src="https://hosted.weblate.org/widgets/newpipe/-/svg-badge.svg"></a>
+<a href="https://web.libera.chat/#newpipe" alt="IRC channel: #newpipe"><img src="https://img.shields.io/badge/IRC%20chat-%23newpipe-brightgreen.svg"></a>
+<a href="https://www.bountysource.com/teams/newpipe" alt="Bountysource bounties"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f"></a>
+</p>
+<hr>
+<p align="center"><a href="#screenshots">Screenshots</a> &bull; <a href="#description">Description</a> &bull; <a href="#features">Fuktionen</a> &bull; <a href="#installation-and-updates">Installation und Aktualisierungen</a> &bull; <a href="#contribution">Beitrag</a> &bull; <a href="#donate">Spende</a> &bull; <a href="#license">Lizens</a></p>
```
Comment (litetex) [inline doc/README.de.md:16]:
```suggestion
<p align="center"><a href="#screenshots">Screenshots</a> &bull; <a href="#description">Beschreibung</a> &bull; <a href="#features">Funktionen</a> &bull; <a href="#installation-and-updates">Installation und Aktualisierungen</a> &bull; <a href="#contribution">Beitrag</a> &bull; <a href="#donate">Spende</a> &bull; <a href="#license">Lizenz</a></p>
```

File: doc/README.de.md [hunk 13]
```diff
@@ -0,0 +1,146 @@
+<p align="center"><a href="https://newpipe.net"><img src="assets/new_pipe_icon_5.png" width="150"></a></p> 
+<h2 align="center"><b>NewPipe</b></h2>
+<h4 align="center">Ein freies, leichtes Streaming-Frontend für Android.</h4>
+
+<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-en.svg" alt="Get it on F-Droid" height=80/></a></p>
```
Comment (litetex) [inline doc/README.de.md:5]: Use the german version of the image please:
```suggestion
<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-de.svg" alt="Get it on F-Droid" height=80/></a></p>
```

File: doc/README.de.md [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+ 2. Lade die APK von [GitHub Releases](https://github.com/TeamNewPipe/NewPipe/releases) herunter und installiere sie.
+ 3. Aktualisierungen mit F-Droid. Dies ist die langsamste von allen Methoden, da F-Droid die Änderungen erkennen, die APK von selbst erstellen, unterzeichnen, und dann für die Benutzer veröffentlichen muss.
+ 4. Erstelle eine Debug-APK für dich. Dies ist die schnellste Methode, um neue Aktualisierungen zu bekommen, aber auch die komplizierteste. Wir empfehlen daher eine der anderen Methoden.
+
+Wir empfehlen Methode 1 für die meisten Nutzer. APKs die mit Methode 1 oder 2 installiert wurden sind miteinander kompatibel, aber nicht mit denen von Methode 3. Dies liegt daran, dass derselbe Signaturschlüssel (unserer) für 1 und 2 verwendet wird, aber ein anderer Signaturschlüssel (F-Droid) für 3 verwendet wird. Beim Erstellen einer Debug-APK mit Methode 4 wird ein Schlüssel vollständig ausgeschlossen. Signaturschlüssel tragen dazu bei, dass ein Benutzer nicht dazu verleitet wird, ein schädliches Update für eine App zu installieren.
```
Comment (litetex) [inline doc/README.de.md:95]:
```suggestion
Wir empfehlen Methode 1 für die meisten Nutzer. APKs die mit Methode 1 oder 2 installiert wurden sind miteinander kompatibel, aber nicht mit denen von Methode 3. Dies liegt daran, dass derselbe Signaturschlüssel (unserer) für 1 und 2 verwendet wird, aber ein anderer Signaturschlüssel (F-Droid) für 3 verwendet wird. Beim Erstellen einer Debug-APK mit Methode 4 wird ein Schlüssel vollständig ausgeschlossen. Signaturschlüssel tragen dazu bei, dass ein Benutzer nicht dazu verleitet wird ein schädliches Update für eine App zu installieren.
```

File: doc/README.de.md [hunk 15]
```diff
@@ -0,0 +1,146 @@
...
+
+Wenn Sie in der Zwischenzeit aus irgendeinem Grund die Quellen wechseln möchten (z. B. die Kernfunktionalität von NewPipe bricht und F-Droid hat noch nicht das neueste Update), empfehlen wir Ihnen, dieses Verfahren zu befolgen:
```
Comment (litetex) [inline doc/README.de.md:97]:
```suggestion
Wenn du in der Zwischenzeit aus irgendeinem Grund die Quellen wechseln möchten (z. B. NewPipe funktioniert nicht korrekt und F-Droid hat noch nicht das neueste Update), empfehlen wir, dieses Verfahren zu befolgen:
```

File: doc/README.de.md [hunk 16]
```diff
@@ -0,0 +1,146 @@
...
+1. Sichere deine Daten in den Einstellungen > Inhalt > Datenbank exportieren, damit deine Abos, Playlists etc. nicht verloren gehen. 
+2. Deinstalliere NewPipe
+3. Lade die APK aus der neuen Quelle herunter und installiere diese
+4. Importiere die Daten aus Schritt 1 via Einstellungen > Inhalt > Datenbank importieren
+
+## Mitmachen
+Egal, ob du Ideen, Übersetzungen, Designänderungen, Codebereinigung oder wirklich schwerwiegende Codeänderungen hast, Hilfe ist immer willkommen.
+Je mehr getan wird, desto besser wird es!
```
Comment (litetex) [inline doc/README.de.md:105]:
```suggestion
Egal ob du Ideen, Übersetzungen, Designänderungen, Codebereinigung oder wirklich schwerwiegende Codeänderungen hast: Hilfe ist immer willkommen. Je mehr getan wird, desto besser wird es!
```

File: doc/README.de.md [hunk 17]
```diff
@@ -0,0 +1,146 @@
...
+
+Wenn du mithelfen möchtest, lese die [contribution notes](.github/CONTRIBUTING.md).
+
+<a href="https://hosted.weblate.org/engage/newpipe/">
+<img src="https://hosted.weblate.org/widgets/newpipe/-/287x66-grey.png" alt="Translation status" />
+</a>
+
+## Spende
+Wenn du NewPipe magst, würden wir uns über eine Spende freuen. Du kannst entweder Bitcoin schicken oder über Bountysource oder Liberapay spenden. Für weitere Informationen über Spenden an NewPipe besuche bitte unsere [Website](https://newpipe.net/donate).
```
Comment (litetex) [inline doc/README.de.md:114]:
```suggestion
Wenn du NewPipe magst, würden wir uns über eine Spende freuen. Du kannst entweder Bitcoin senden oder über Bountysource oder Liberapay spenden. Für weitere Informationen über Spenden an NewPipe besuche bitte unsere [Website](https://newpipe.net/donate).
```

File: doc/README.de.md [hunk 18]
```diff
@@ -0,0 +1,146 @@
...
+Daher sammelt die App keine Daten ohne Ihre Zustimmung. In der Datenschutzerklärung von NewPipe wird detailliert erläutert, welche Daten gesendet und gespeichert werden, wenn Sie einen Unfallbericht senden oder in unserem Blog kommentieren. Sie können das Dokument [hier](https://newpipe.net/legal/privacy/) finden.
+
+## Lizens
+[![GNU GPLv3 Image](https://www.gnu.org/graphics/gplv3-127x51.png)](https://www.gnu.org/licenses/gpl-3.0.en.html)  
+
+NewPipe ist eine freie Software: Du kannst sie benutzen, studieren, teilen, und verbessern.
+Insbesondere können Sie es weitergeben und/oder modifizieren unter den Bedingungen der
+[GNU General Public License](https://www.gnu.org/licenses/gpl.html) wie
+veröffentlicht von der Free Software Foundation, entweder in Version 3 der Lizenz, oder
+(nach Ihrer Wahl) jede spätere Version.
```
Comment (litetex) [inline doc/README.de.md:146]:
```suggestion
NewPipe ist eine freie Software: Du kannst sie benutzen, studieren, teilen, und verbessern wie du magst. Insbesondere kannst du es weitergeben und/oder modifizieren unter den Bedingungen der [GNU General Public License](https://www.gnu.org/licenses/gpl.html) wie veröffentlicht von der Free Software Foundation, entweder in Version 3 der Lizenz, oder (nach Ihrer Wahl) jede spätere Version. 
```

File: doc/README.de.md [hunk 19]
```diff
@@ -0,0 +1,146 @@
...
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="assets/bountysource_qr_code.png" alt="Visit NewPipe at bountysource.com" width="100px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="Check out how many bounties you can earn."></a></td>
+  </tr>
+</table>
+
+## Datenschutzbestimmungen
+
+Das NewPipe-Projekt zielt darauf ab, eine private, anonyme Erfahrung bei der Nutzung von Medien-Webdiensten zu ermöglichen.
+Daher sammelt die App keine Daten ohne Ihre Zustimmung. In der Datenschutzerklärung von NewPipe wird detailliert erläutert, welche Daten gesendet und gespeichert werden, wenn Sie einen Unfallbericht senden oder in unserem Blog kommentieren. Sie können das Dokument [hier](https://newpipe.net/legal/privacy/) finden.
```
Comment (litetex) [inline doc/README.de.md:137]:
```suggestion
Das NewPipe-Projekt zielt darauf ab eine private, anonyme Erfahrung bei der Nutzung von Medien-Webdiensten zu ermöglichen. Daher sammelt die App keine Daten ohne Ihre Zustimmung. In der Datenschutzerklärung von NewPipe wird detailliert erläutert, welche Daten gesendet und gespeichert werden, wenn du einen Fehlerbericht sendest oder in unserem Blog kommentierst kannst du [hier](https://newpipe.net/legal/privacy/) finden.
```

File: doc/README.de.md [hunk 20]
```diff
@@ -0,0 +1,146 @@
...
+
+### Funktionen
+
+* Durchsuche YouTube nach Videos
+* Keine Anmeldung nötig
+* Allgemeine Informationen zu Videos
+* Schaue YouTube Videos
+* Höre Musik auf YouTube
+* Popup-Modus (schwebendes Video)
+* Wählen Sie den Streaming-Player aus, mit dem Sie Videos ansehen möchten
```
Comment (litetex) [inline doc/README.de.md:53]:
```suggestion
* Wähle selber den Streaming-Player aus, mit dem du Videos ansehen möchtest
```

File: doc/README.de.md [hunk 21]
```diff
@@ -0,0 +1,146 @@
...
+<a href="https://www.bountysource.com/teams/newpipe" alt="Bountysource bounties"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f"></a>
+</p>
+<hr>
+<p align="center"><a href="#screenshots">Screenshots</a> &bull; <a href="#description">Description</a> &bull; <a href="#features">Fuktionen</a> &bull; <a href="#installation-and-updates">Installation und Aktualisierungen</a> &bull; <a href="#contribution">Beitrag</a> &bull; <a href="#donate">Spende</a> &bull; <a href="#license">Lizens</a></p>
+<p align="center"><a href="https://newpipe.net">Webseite</a> &bull; <a href="https://newpipe.net/blog/">Blog</a> &bull; <a href="https://newpipe.net/FAQ/">FAQ</a> &bull; <a href="https://newpipe.net/press/">Presse</a></p>
+<hr>
+
+*Lese dieses Dokument in anderen Sprachen: [English](README.md), [Deutsch](doc/README.de.md), [Español](doc/README.es.md), [한국어](doc/README.ko.md), [Soomaali](doc/README.so.md), [Português Brasil](doc/README.pt_BR.md), [Polski](doc/README.pl.md), [日本語](doc/README.ja.md), [Română](doc/README.ro.md), [Türkçe](doc/README.tr.md), [正體中文](doc/README.zh_TW.md).*
```
Comment (litetex) [inline doc/README.de.md:20]: I think this should be moved to a new line (and centered) as there are too many languages in one line

**Review by litetex — CHANGES_REQUESTED** [review]
Thank you for the PR.

Found these major problems:
* Images are not loading/link are wrong
* Nearly every line contains a spelling mistake
* The other Readme's need references to this one

**Comment by litetex** [comment]
No feedback since over 1 month :/

Closing this for now.

**Comment by lebogg** [comment]
I already started with a translation as well and now saw this closed pull request. I would finish it while also incorporating your comments on this previous attempt @litetex ?

**Comment by opusforlife2** [comment]
@lebogg You're most welcome to. :D

**Pull request closed without merging**